packages/@react-aria/interactions/test/DOMPropsResponder.test.js

```diff
@@ -0,0 +1,61 @@
+import {cleanup, fireEvent, render} from '@testing-library/react';
+import {DOMPropsResponder, Hoverable} from '../';
+import React from 'react';
+
+describe('DOMPropsResponder', function () {
+  afterEach(cleanup);
+
+  it('should handle hover events on nested hoverable children', function () {
+    let onHover = jest.fn();
+    let {getByRole} = render(
+      <DOMPropsResponder>
+        <div>
+          <Hoverable onHover={onHover}>
+            <button>Button</button>
+          </Hoverable>
+        </div>
+      </DOMPropsResponder>
```

Comment:
Ah I think I understand what you are trying to test here now, these are like the PressResponder tests yeah? You should probably move the `onHover` and `ref` props to `DOMPropsResponder` so it actually tests that the `DOMPropsResponder` context is being set up properly and being consumed by `Hoverable`.

```
<DOMPropsResponder onHover={onHover}>
....
```